### PR TITLE
Feature/3653

### PR DESCRIPTION
**Pull request by cyrilkyburz**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
 import hmac
 import os
 import requests
 
-app = Flask(__name__)
-app.debug = os.getenv('DEBUG', '') == 'True'
+def create_app():
+    app = Flask(__name__)
+    app.debug = os.getenv('DEBUG', '') == 'True'
+    if os.getenv('SENTRY_DSN', ''):
```
Comment (lukaselmer) [inline server.py]: hmm, i guess we don't need the `, '')` part?
https://docs.python.org/3/library/os.html#os.getenv
so i suggest

``` python
    app.debug = os.getenv('DEBUG') == 'True'
    if os.getenv('SENTRY_DSN'):
```

File: server.py [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
+        sentry = Sentry()
+        sentry.init_app(app)
+    return app
 
+app = create_app()
```
Comment (lukaselmer) [inline server.py:17]: nice refactoring 👍

**Comment by lukaselmer** [comment]
LGTM!

File: server.py (hunk 1 above)
Comment (cyrilkyburz) [inline server.py]: you are right

**Pull request merged**